vectordatasource/transform.py

```diff
@@ -1608,10 +1608,10 @@ def intracut(ctx):
     'province': 4,
     'state': 4,
 
-    'sea': 3,
+    'sea': 4,
```

Comment:
These are junk DNA at this point? But now at least they are consistent with everything else, and this section does occasionally get triggered for other places, so didn't feel comfortable removing totally (even just for ocean and sea).